health/main.py

```diff
@@ -20,15 +20,31 @@
 from health.api.v1 import health
 from health.api.v1 import regions
 
-app = flask.Flask(__name__, static_folder=None)
-config_path = os.environ.get("HEALTH_CONF", "/etc/health/config.json")
 
-try:
-    config = json.load(open(config_path))
-    app.config.update(config)
-except IOError as e:
-    logging.warning("Config at '%s': %s" % (config_path, e))
+CONF = None
+DEFAULT_CONF = {
+    "flask": {
+        "PORT": 5000,
+        "HOST": "0.0.0.0",
+        "DEBUG": true
+    }
+}
+
+
+if not CONF:
+    path = os.environ.get("HEALTH_CONF", "/etc/health/config.json")
+    try:
+        CONF = json.load(open(path))
+        logging.info("Config is '%s'" % path)
+    except IOError as e:
+        logging.warning("Config at '%s': %s" % (config_path, e))
+        CONF = DEFAULT_CONF
 
+APP_CONF = CONF["flask"]
```

Comment:
so if there is no flask section in /etc/health/config.json — main.py would fail. I think we want to have defaults for everything